content/docs/command-reference/exp/show.md

```diff
@@ -74,7 +74,12 @@ will be generated using the same data from the table.
 - `-A`, `--all-commits` - include experiments derived from all Git commits, as
   well as from the last one. This prints all experiments in the project.
 
-- `-n <num>`, `--num <num>` - show the last `num` commits from HEAD.
+- `--rev <commit>` - show experiments derived from the specified `<commit>` as
+  baseline. Defaults to `HEAD` if none of `--rev`, `-a`, `-T`, `-A` is used.
+
+- `-n <num>`, `--num <num>` - show experiments from the previous `num` commits
+  (before the `--rev` baseline). For example `-n -2` includes experiments from
+  `HEAD` and its 2 first ancestors.
```

Comment:
The info about negative values seems wrong here after all (sorry I was confused). The text should be closer to https://github.com/iterative/dvc/pull/7245#pullrequestreview-902522092, right? Please double check the suggestion, thanks!

```suggestion
- `-n <num>`, `--num <num>` - show experiments from the `--rev` baseline and
  from `num` commits before it (first parents). Give a negative value to include
  all first-parent commits (similar to `git log -n`).
```

Reply:
But TBH I'm not sure why this is being changed here if there's no change for `--num` in https://github.com/iterative/dvc/pull/7204/files 🤷🏼

Reply:
Does this PR also depend on https://github.com/iterative/dvc/pull/7245/files ? Marked as waiting for core merge for now.

Reply:
> But TBH I'm not sure why this is being changed here if there's no change for `--num` in the related core PR, [iterative/dvc#7204](https://github.com/iterative/dvc/pull/7204) 🤷🏼

I just forgot to update the doc in https://github.com/iterative/dvc/pull/7204.